### PR TITLE
Create unwrapRefs function

### PR DESCRIPTION
**Pull request by leroykorterink**

### CODE REVIEW

File: src/unwrapRefs/unwrapRefs.stories.mdx [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+```ts
+export function unwrapRefs<T extends Refs>(refs: T): UnwrapRefs<T>;
+```
+
+### Returns
+
+Object with unwrapped RefObjects
+
+## Usage
```
Comment (ThaNarie) [inline src/unwrapRefs/unwrapRefs.stories.mdx:19]: Please copy fixed version of example from the other PR.

**Review by ThaNarie — CHANGES_REQUESTED** [review]

File: src/unwrapRefs/unwrapRefs.ts [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+ * Unwraps RefObjects in object
+ */
+export function unwrapRefs<T extends Refs>(refs: T): UnwrapRefs<T> {
+  const unwrappedRefs = {} as UnwrapRefs<T>;
+
+  for (const key in refs) {
+    if (Object.hasOwn(refs, key)) {
+      unwrappedRefs[key] = refs[key].current;
+    }
+  }
```
[the remark this answers is not in the document]
Comment (leroykorterink) [reply src/unwrapRefs/unwrapRefs.ts:14]: You're losing the key type with `Object.keys` (it'll just be a `string`) so chose to write it out like this. Typescript maintains the type of `key` with `for in` (`Extract<keyof T, string>`), the `Object.hasOwn` is enforced by `eslint` when `for in` is used.

https://eslint.org/docs/latest/rules/guard-for-in

[the remark this answers is not in the document]
Comment (ThaNarie) [reply src/unwrapRefs/unwrapRefs.ts:14]: Ah yes, the types 😄

File: src/unwrapRefs/unwrapRefs.stories.mdx [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+}>;
+
+function DemoComponent() {
+  const refs = useRef<DemoComponentRefs>();
+
+  useEffect(() => {
+    const { element, elements } = unwrapRefs(refs);
+
+    console.log(element); // HTMLHeadingElement | null
+    console.log(elements); // Array<HTMLButtonElement | null> | null
```
Comment (ThaNarie) [inline src/unwrapRefs/unwrapRefs.stories.mdx:34]:
```suggestion
    const { heading, buttons } = unwrapRefs(refs);

    console.log(heading); // HTMLHeadingElement | null
    console.log(buttons); // Array<HTMLButtonElement | null> | null
```

**Review by ThaNarie — CHANGES_REQUESTED** [review]

**Review by ThaNarie — APPROVED** [review]

**Pull request merged**